frontends/web/src/locales/en/app.json

```diff
@@ -793,6 +793,10 @@
       }
     },
     "settings": {
+      "sats": {
+        "text": "A Satoshi ('sats' for short) is the smallest unit of Bitcoin. One Satoshi is a hundred millionth of a bitcoin (0.00000001 BTC). It is named after the creator of Bitcoin, Satoshi Nakamoto.",
```

Comment:
Maybe better to use 'sat' instead of 'sats', since it is the abbreviation used everywhere in the app?

Reply:
fixed thanks